packages/airnode-node/src/api/index.ts

```diff
@@ -198,7 +209,10 @@ export async function performApiCall(
   if (!goRes.success) {
     const { aggregatedApiCall } = payload;
     const log = logger.pend('ERROR', `Failed to call Endpoint:${aggregatedApiCall.endpointName}`, goRes.error);
-    return [[log], { success: false, errorMessage: `${RequestErrorMessage.ApiCallFailed}` }];
+    // eslint-disable-next-line import/no-named-as-default-member
+    const axiosErrorMsg = axios.isAxiosError(goRes.error) ? errorMsgFromAxiosError(goRes.error) : '';
+    const errorMessage = [RequestErrorMessage.ApiCallFailed, axiosErrorMsg].filter(Boolean).join(' ');
```

Comment:
You can use [lodash compact](https://lodash.com/docs/4.17.15#compact) instead of `filter(Boolean)`. It's imo more readable but I'll leave it up to you.

Reply:
Thanks and will do 👍